crates/builder/src/tasks/submit.rs

```diff
@@ -112,6 +87,15 @@ impl SubmitTask {
         Ok(TransactionRequest::default().with_blob_sidecar(sidecar).with_input(data))
     }
 
+    async fn next_host_block_height(&self, ru_chain_id: U256) -> eyre::Result<U256> {
+        let result = self.zenith.lastSubmittedAtBlock(ru_chain_id).call().await?;
```

Comment:
instead of asking the contract, we just want to ask the provider what the current block number is.

`self.provider.get_block_number().await?.checked_add(1).ok_or_else(|| ...)`